index.d.ts

```diff
@@ -102,6 +102,7 @@ declare namespace execa {
 
 		/**
 		Keep track of the spawned process and `kill` it when the parent process exits.
+		Kill the spawned process when the parent process exits unless either the spawned process is detached, or the parent process is terminated abruptly.
```

Comment:
The TS text should be exactly what's in the readme.

Reply:
Sure, I will copy it.
I was thinking TypeScript comments appear in VS Code "suggestions" as you type, i.e. are read in a different context than a `README` and might need to be more concise. 
But copying the `README` makes it simpler to maintain.